scripts/tests

```diff
@@ -1379,6 +1417,13 @@ def cli():
         parents=[common_parser],
     )
 
+    # alloc arguments
+    alloc_parser = cmd_subparsers.add_parser(
+        "alloc",
+        help="Run the alloc tests for all parameter sets",
+        parents=[common_parser],
+    )
```

Comment:
To ease maintenance, please reduce the diff to the mlkem-native tests script.

Reply:
Done